src/products/ProductMapper.js

```diff
@@ -119,10 +119,15 @@ class ProductMapper {
             variants = product.variants.map(v => this._mapProductVariant(product, v.product));
             masterVariantId = variants[0].sku;
         } else {
-            variants = [{
-                id: product.sku,
-                sku: product.sku
-            }];
+            variants = [
+                new ProductVariant.Builder()
+                    .withAvailable(true) // TODO: Get actual value from backend
+                    .withId(product.sku)
+                    .withName(product.name || '')
+                    .withPrices([])
```

Comment:
Why not set the prices too?

Reply:
The variant doesn't have a dedicated price, this is a "fake" variant that we have to set because our model mandates that a product always has a variant.

Reply:
The price is set for real variants so for consistency I think we should set the price for a 'fake' variant too from the product. If our model mandates that every product always has a variant and we guarantee that then there is no more distinction between fake variant and real variant for the API client.

Reply:
Yes ok you're right, if we set other fields, let's also do it for prices to be consistent. I just committed that change.